docs/reference-guides/data/data-core-blocks.md

```diff
@@ -367,9 +825,37 @@ _Returns_
 
 Returns an action object used to set block categories.
 
+_Usage_
+
+```js
+import { store as blocksStore } from '@wordpress/blocks';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+const ExampleComponent = () => {
+	const blockCategories = useSelect( ( select ) =>
+		select( blocksStore ).getCategories()
+	);
```

Comment:
```suggestion
		select( blocksStore ).getCategories(),
		[]
	);
```

Reply:
Ah! Yes, I added that to the actions.js and didn't update it. Updated now.